app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_two_parameters_equal_to_zero() -> None:
+    assert (get_human_age(0, 0) == [0, 0])
+
+
+def test_one_parameter_equal_to_zero() -> None:
+    assert (get_human_age(0, 21) == [0, 1])
+
+
+def test_float_value_of_parameters() -> None:
+    assert (get_human_age(23.2, 25) == [1, 2])
```

Comment:
The test `test_float_value_of_parameters` is checking for float inputs. Ensure that the function `get_human_age` is expected to handle float inputs, as this might not be a typical use case for age calculations.